package/Resources/Translations/zh-Hans/Translation.ini

```
@@ -0,0 +1,1704 @@
+Client:Main:ChatboxCommandProtocolVersionInvalid=ProtocolVersion 只允许值 0 和 2。; ProtocolVersion only allows values 0 and 2.
+Client:Main:ChatboxCommandProtocolVersionSyntax=命令语法：/ProtocolVersion <数字>。; Command syntax: /ProtocolVersion <number>.
+Client:Main:ChatboxCommandRandomStartsHelp=启用完全随机的起始位置 (仅限基于泰伯利亚之日的游戏)。; Enables completely random starting locations (Tiberian Sun based games only).
+Client:Main:ChatboxCommandRollHelp=掷骰子，例如 /roll 3d6; Roll dice, for example /roll 3d6
+Client:Main:ChatboxCommandRollInvalid2=一次只能掷 1 到 10 个骰子。; You can only between 1 to 10 dies at once.
+Client:Main:ChatboxCommandRollInvalid3=骰子只能有 2 到 100 个面。; You can only have between 2 and 100 sides in a die.
+Client:Main:ChatboxCommandRollInvalidAndSyntax=指定的骰子无效。预期格式：/roll <骰子数量>d<骰子面数>; Invalid dice specified. Expected format: /roll <die count>d<die sides>
+Client:Main:ChatboxCommandSaveOptionsHelp=保存游戏选项预设，以便日后加载; Save game option preset so it can be loaded later
+Client:Main:ChatboxCommandShowMapsHelp=显示地图列表 (仅限游戏主机); Show map list (game host only)
+Client:Main:ChatboxCommandTipText=要使用命令，请以 /<command> 开始您的消息。可能的聊天框命令：; To use a command, start your message with /<command>. Possible chat box commands:
+Client:Main:ChatboxCommandTipTitle=聊天框命令帮助; Chat Box Command Help
+Client:Main:ChatCommandTip=输入 / 查看可用聊天命令列表。; Type / to view a list of available chat commands.
+Client:Main:ChatHere=在此输入聊天内容...; Type here to chat...
+Client:Main:Cheater=作弊者！; CHEATER!
+Client:Main:CheaterText=检测到修改过的游戏文件。它们可能会影响游戏体验。@@难道你真的没有不作弊就赢得任务的本事吗？; Modified game files have been detected. They could affect@the game experience.@@Do you really lack the skill for winning the mission without@cheating?
+Client:Main:CheckingForUpdates=正在检查更新...; Checking for updates...
+Client:Main:CheckUpdateFailedClickToRetry=检查更新失败！点击重试。; Checking for updates failed! Click to retry.
+Client:Main:ChoiceWhatDoYouWant=您想做什么？; What do you want to do?
+Client:Main:ClearStatistics=清除统计数据; Clear Statistics
+Client:Main:ClearStatisticsText=所有统计数据将从数据库中清除。@@您确定要继续吗？; All statistics data will be cleared from the database.@@Are you sure you want to continue?
+Client:Main:ClearStatisticsTitle=清除所有统计数据; Clear all statistics
+Client:Main:ClickToCheckUpdate=点击检查更新。; Click to check for updates.
+Client:Main:ClientDisconnectedAfterRetries=达到最大连接重试次数后，已从 CnCNet 断开。; Disconnected from CnCNet after reaching the maximum number of connection retries.
+Client:Main:ClientGameTypeNotFoundException=客户端配置似乎未迁移以适应 v2.12 的更改。请在 'ClientDefinitions.ini' 文件的 '[Settings]' 部分指定 'ClientGameType' (允许的选项: {0})。@@有关更多详细信息，请参阅客户端 {1} 的文档。此链接也可以在日志文件中找到。; It seems the client configuration was not migrated to accommodate for the v2.12 changes. Please specify 'ClientGameType' in '[Settings]' section of the 'ClientDefinitions.ini' file (allowed options: {0}).@@Please refer to documentation of the client {1} for more details. This link can also be found in the log file.
```

`客户端配置似乎未迁移以适应 v2.12 的更改`: even an AI can give you the correct suggestion to make the Chinese translation smoth.

`{1}` here acts as a placeholder for a URL. So `客户端 {1} 的文档` is incorrect.

```suggestion
Client:Main:ClientGameTypeNotFoundException=客户端配置似乎未适配 v2.12 版本的变更。请在 'ClientDefinitions.ini' 文件的 '[Settings]' 部分指定 'ClientGameType' (允许的选项: {0})。@@有关更多详细信息，请参阅客户端的文档  {1} 。此链接也可以在日志文件中找到。; It seems the client configuration was not migrated to accommodate for the v2.12 changes. Please specify 'ClientGameType' in '[Settings]' section of the 'ClientDefinitions.ini' file (allowed options: {0}).@@Please refer to documentation of the client {1} for more details. This link can also be found in the log file.
```

---

###✅ Done